de.metas.adempiere.adempiere/base/src/main/java-legacy/org/compiere/util/TimeUtil.java

```diff
@@ -1839,4 +1839,50 @@ public static ZonedDateTime convertToTimeZone(@NonNull final ZonedDateTime date,
 			return date.toInstant().atZone(zoneId);
 		}
 	}
+
+	/**
+	 * Counterpart of {@link #deserializeInstant(String)}.
+	 */
+	public static String serializeInstant(@NonNull final Instant instant)
+	{
+		return Long.toString(instant.getEpochSecond()) + "." + Long.toString(instant.getNano());
+	}
+
+	/**
+	 * Deserializes a string such as {@code "12345.234567"} into an {@link Instant}.
+	 * <p>
+	 * Notes:
+	 * <li>I didn't want to use jackson because this is not inteded to serialize/deserialize a whole object-tree, but that to play a role in very concrete business logic, so i want it to be more transparent how this is done.
```

Comment:
```suggestion
	 * <li>I didn't want to use jackson because this is not inteded to serialize/deserialize a whole object-tree, but be called from very concrete business logic, so i want it to be more transparent how this is done.
```